official/utils/logs/logger_test.py

```diff
@@ -31,16 +31,58 @@
 
 class BenchmarkLoggerTest(tf.test.TestCase):
 
+  def test_get_default_benchmark_logger(self):
+    self.assertIsInstance(logger.get_benchmark_logger(),
+                          logger.BaseBenchmarkLogger)
+
+  def test_config_base_benchmark_logger(self):
+    logger.config_benchmark_logger("")
+    self.assertIsInstance(logger.get_benchmark_logger(),
+                          logger.BaseBenchmarkLogger)
+
+  def test_config_benchmark_file_logger(self):
+    logger.config_benchmark_logger("/tmp/abc")
+    self.assertIsInstance(logger.get_benchmark_logger(),
+                          logger.BenchmarkFileLogger)
```

Comment:
Can we test that the lock is acquired as well? Don't know how feasible that is in a unit test, but would be nice to make sure we can't end up with two loggers.

Reply:
locking is hard to test, which usually involve some fuzzy test, like create instance thousands of times, and hope can catch a error, which usually hard to do in a unit test.